src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Main {
+    private static final int ARRAY_SIZE = 6;
+
+    public static void main(String[] args) {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the empty line above the `main` method declaration.